app/javascript/v3/api/auth.js

```diff
@@ -42,6 +42,19 @@ export const register = async creds => {
   return null;
 };
 
+export const accountSetup = async ({ id, name, locale }) => {
+  try {
+    const response = await wootAPI.put(`api/v2/accounts/${id}.json`, {
```

Comment:
Why do we have `.json` added here? Is there a specific reason?

Reply:
Removed the function as we are not using it.